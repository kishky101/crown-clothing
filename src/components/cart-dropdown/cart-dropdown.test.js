import renderer from 'react-test-renderer';
import CartDropdown from './cart-dropdown.component';
import { MockStore } from '../../utils/test/setup';
import { BrowserRouter as Router } from 'react-router-dom';


test('creating cart-dropdown snapshot', () => {
    const cartdropdown = renderer.create(
        <MockStore>
            <Router>
                <CartDropdown />
            </Router>
        </MockStore>
    ).toJSON;
    expect(cartdropdown).toMatchSnapshot();
})
