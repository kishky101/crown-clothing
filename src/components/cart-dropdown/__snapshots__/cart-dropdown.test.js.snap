// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating cart-dropdown snapshot 1`] = `[Function]`;
